perfkitbenchmarker/linux_packages/hadoop.py

```diff
@@ -69,7 +69,10 @@ def YumInstall(vm):
 
 def AptInstall(vm):
   """Installs Hadoop on the VM."""
-  vm.InstallPackages('libsnappy1 libsnappy-dev')
+  libsnappy = 'libsnappy1'
+  if not vm.HasPackage(libsnappy):
+    libsnappy = 'libsnappy1v5'
```

Comment:
libsnappy1v5 is the package name on Ubuntu 16.04? A comment there would be nice.